services/kuber/templates/cluster-autoscaler.goyaml

```diff
@@ -3,6 +3,9 @@ apiVersion: v1
 kind: ConfigMap
 metadata:
   name: autoscaler-config-{{ .ClusterID }}
+  labels:
+    app.kubernetes.io/part-of: claudie
+    app.kubernetes.io/name: cluster-autoscaler-{{ .ClusterID }}
```

Comment:
I think this shouldn't be templated, I'd go with `app.kubernetes.io/name: cluster-autoscaler` for everything

Reply:
There are 2 deployments. One for cluster-autoscaler and one for autoscaler-adapter. If we use `app.kubernetes.io/name: cluster-autoscaler` label on both of them then the deployments will also select each others pods. We need a separate label for each of them, differentiating the pods of cluster-autoscaler and autoscaler-adapter.

Reply:
Thanks, I agree. I suggest https://github.com/berops/claudie/pull/714/files#r1176620860